pkg/apis/serving/v1beta1/predictor_tfserving.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2020 kubeflow.org.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"fmt"
+	"github.com/kubeflow/kfserving/pkg/constants"
+	v1 "k8s.io/api/core/v1"
+	"strings"
+)
+
+var (
+	TensorflowEntrypointCommand          = "/usr/bin/tensorflow_model_server"
+	TensorflowServingGRPCPort            = "9000"
+	TensorflowServingRestPort            = "8080"
+	TensorflowServingGPUSuffix           = "-gpu"
+	InvalidTensorflowRuntimeVersionError = "Tensorflow RuntimeVersion must be one of %s"
+	InvalidTensorflowRuntimeIncludesGPU  = "Tensorflow RuntimeVersion is not GPU enabled but GPU resources are requested. " + InvalidTensorflowRuntimeVersionError
+	InvalidTensorflowRuntimeExcludesGPU  = "Tensorflow RuntimeVersion is GPU enabled but GPU resources are not requested. " + InvalidTensorflowRuntimeVersionError
+)
+
+// TFServingSpec defines arguments for configuring Tensorflow model serving.
+type TFServingSpec struct {
```

Comment:
After our discussion on frameworks, I think we decided to go back to calling this Tensorflow instead of TFServing. The same way TorchServe will be called PyTorch.

Reply:
the name on yaml is `tensorflow` but I'd like to keep internal struct still as `TFServingSpec` as it is really creating tfserving container under the hood.